pullrequests/setenv.sh

```diff
@@ -0,0 +1,10 @@
+export HYDRACTL_USERNAME="hydra"
+export HYDRACTL_PASSWORD="passwordko"
```

Comment:
Passwords are not allowed in our repos.

Reply:
Totallly imaginery password in the example settings file...I might add just a comment "set this variable to hold your admin password"?

Reply:
Left messages user to set up admin passwd manually in shell, in order to prevent accidental commits to public repo